.github/workflows/nightly-windows-ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Fast-DDS Windows CI
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  nightly-sec-windows-ci:
+    uses: ./.github/workflows/reusable-windows-ci.yml
+    with:
+      label: 'nightly-sec-windows-ci'
+      cmake-args: "-DSECURITY=ON"
+      ctest-args: "-LE xfail"
+      fastdds_branch: 'master'
```

Comment:
I think it does not make sense to run nightly CI on `master` here. Should we change it into `2.6.x`?